models/rule_helpers.go

```diff
@@ -192,6 +193,9 @@ func ChangeRule(q *reform.Querier, ruleID string, params *ChangeRuleParams) (*Ru
 	row.Severity = params.Severity
 	row.Filters = params.Filters
 	row.Params = params.RuleParams
+	if params.Summary != "" {
```

Comment:
I'm not sure about this one.
Everything else is set from request.
But If I do it here, it will be breaking change.

Reply:
it's interesting case. In all other endpoints we change only fields that we get from request, but doesn't change other ones. And we should keep this behaviour.
IA is still private API, so you can change behaviour.